src/components/toolSelfCheckResponse/index.tsx

```diff
@@ -0,0 +1,145 @@
+// import React, { useState } from "react";
+import { AddIcon } from "@chakra-ui/icons";
+import { Box, Button, Checkbox, Input, Textarea, Text } from "@chakra-ui/react";
+
+//Interface Props
+export interface SelfCheckResponseCardProps {
+    key: React.Key;
+    breakpointNum: number;
+    lowerBound: number;
+    upperBound: number;
+    setUpperbound: (upper: number) => void;
+    lastBreakpoint: boolean;
+    breakpointAfter?: boolean;
+    setLastBreakpoint: (last: boolean) => void;
+    description: string;
+    setDescription: (desc: string) => void;
+    addNewBreakpoint: () => void;
+    secondaryDesc?: string;
+    setSecondaryDesc?: (desc: string) => void;
+}
+
+//Self check response card component
+export const SelfCheckResponseCard: React.FC<SelfCheckResponseCardProps> = ({
+    breakpointNum,
+    lowerBound,
+    upperBound,
+    setUpperbound,
+    lastBreakpoint,
+    breakpointAfter = false,
+    setLastBreakpoint,
+    description,
+    setDescription,
+    addNewBreakpoint,
+    secondaryDesc = "",
+    setSecondaryDesc = undefined,
+}) => {
+    return (
+        <Box borderRadius="lg" rounded="md" bg="gray.50" pt={10} px={10} mb={8}>
+            <Text fontSize={16} fontWeight={700} mr={6}>
+                Set Breakpoint Score {breakpointNum}
+            </Text>
+            <Text>
+                If users score between these two values, what will you tell
+                them?
+            </Text>
+            <Box pb={5} display="inline-block">
+                <Input
+                    isDisabled
+                    placeholder={String(lowerBound)}
+                    textAlign="center"
+                    width="35%"
+                    display="inline-block"
+                />
+                <Text
+                    fontSize={16}
+                    fontWeight={700}
+                    mx={6}
+                    display="inline-block"
+                >
+                    to
+                </Text>
+                <Input
+                    bg="whiteAlpha"
+                    placeholder="Breakpoint Score"
+                    value={upperBound}
+                    textAlign="center"
+                    width="35%"
+                    display="inline-block"
+                    onBlur={(str) =>
```

Comment:
Try changing `onBlur` to `onChange` to be able to edit the breakpoint score after unfocusing.